ts-perf/packages/commands/src/trace/index.ts

```diff
@@ -278,15 +278,6 @@ const command: Command<TraceOptions> = {
             description: "Saves the trace output to <file>.",
             group: "deoptExplorer",
         },
-        scenarioConfigDirs: {
-            type: "string",
-            longName: "scenarioConfigDir",
-            alias: "scenarioConfigDirs",
-            defaultValue: () => [],
-            param: "directory",
-            multiple: true,
-            description: "Paths to directories containing scenario JSON files.",
-        },
```

Comment:
Thanks for reducing this copy paste; the copy paste was a quick hack to make it work but I never got around to fixing it...